wherehows-kafka/src/main/java/wherehows/utils/JobsUtil.java

```diff
@@ -0,0 +1,114 @@
+/**
```

Comment:
My understanding is we are trying to not rely on wherehows-common. 
Once we refactored all models to wherehows-dao, we can remove the dependency :)
But let me know if you have any concerns.